guides/common/modules/proc_configuring-a-host-for-registration.adoc

```diff
@@ -2,24 +2,42 @@
 = Configuring a host for registration
 
 Configure your host for registration to {ProjectServer} or {SmartProxyServer}.
+You can use a configuration management tool to configure multiple hosts at once.
 
 .Prerequisites
 * The host must be using a supported operating system.
 For more information, see xref:supported-clients-in-registration_{context}[].
+ifdef::satellite,orcharhino,katello[]
+* {ProjectServer} and any {SmartProxyServers} must have a time-synchronization tool enabled and running and be synchronized with the same NTP server.
+For more information, see the following resources:
+** link:{InstallingServerDocURL}synchronizing-the-system-clock-with-chronyd_{project-context}[Synchronizing the system clock with chronyd] in _{InstallingServerDocTitle}_
+ifdef::satellite[]
+** link:{InstallingServerDisconnectedDocURL}synchronizing-the-system-clock-with-chronyd_{project-context}[Synchronizing the system clock with chronyd] in _{InstallingServerDisconnectedDocTitle}_
+endif::[]
+** link:{InstallingSmartProxyDocURL}synchronizing-the-system-clock-with-chronyd_{smart-proxy-context}[Synchronizing the system clock with chronyd] in _{InstallingSmartProxyDocTitle}_
+endif::[]
 
 .Procedure
-* Ensure that a time-synchronization tool is enabled and running on the host.
+. Enable and start a time-synchronization tool on your host.
+The host must be synchronized with the same NTP server as {ProjectServer} and any {SmartProxyServers}.
 ifdef::client-content-dnf[]
-** For {EL} 7 and newer:
+** On {EL} 7 and later:
 +
 ----
 # systemctl enable --now chronyd
 ----
-** For {EL} 6:
+** On {EL} 6:
 +
 ----
 # chkconfig --add ntpd
 # chkconfig ntpd on
 # service ntpd start
 ----
 endif::[]
+. If you use a self-signed {Project} CA certificate, deploy the CA certificate on your host so that the host can make a secured registration call.
+ifdef::managing-hosts[]
+For more information, see xref:deploying-the-ca-certificate-on-a-host-manually[].
+endif::[]
+ifndef::managing-hosts[]
+{ManagingHostsDocURL}deploying-the-ca-certificate-on-a-host-manually[Deploying the CA certificate ona  host manually] in _{ManagingHostsDocTitle}_.
+endif::[]
```

Comment:
@maximiliankolb FYI, this is the step to configure the CA cert. Since there's a procedure how to do that I'm just linking to it instead of duplicating the content.

Reply:
On second thought, I think I'm mixing up two different certs. The original procedure talks about locating the **SSL CA file** setting, which refers to a different cert.